freetextresponse/mixins.py

```diff
@@ -0,0 +1,32 @@
+"""
+Mixins for the Free Text Response XBlock
+"""
+import datetime
+import pytz
+from django.conf import settings
+
+
+class EnforceDueDates(object):
+    """
+    xBlock Mixin to allow xblocks to check the due date
+    (taking the graceperiod into account) of the
+    subsection in which they are placed
+    """
+    # get instance timezone
+    site_tz = pytz.timezone(getattr(settings, 'TIME_ZONE', pytz.utc.zone))
```

Comment:
Please move this to the top-level of the file;
it doesn't need recalculated for each instantiation of the class
(which is basically every page load).

```python
TIME_ZONE = pytz.timezone(getattr(settings, 'TIME_ZONE', pytz.utc.zone))
```